libraries/standard/mqtt/include/mqtt.h

```diff
@@ -123,6 +124,13 @@ struct MQTTContext
     bool waitingForPingResp;
 };
 
+struct MQTTDeserializedInfo
+{
+    uint16_t packetIdentifier;
+    MQTTPublishInfo_t * pPublishInfo;
+    MQTTStatus_t status;
```

Comment:
(Nit) Suggestion to make the meaning of the variable clear
```suggestion
    MQTTStatus_t deserializationStatus;
```

Reply:
I'm unsure about this since the name of the struct is already `MQTTDeserializedInfo`, so I thought it was already clear what the status is referring to. Do you think it still should be changed?

Reply:
Thinking from the customer's perspective, I don't think it may be obvious about what `status` the member of the `MQTTDeserializedInfo` represents. Does it mean a status value returned by the server? or does it mean the status value of our de-serialization effort in the library? 
As this member currently only represents whether the server rejected or accepted the request, I think it would be better to have a more descriptive name like `ackStatus` .

Reply:
Renamed it to `deserializationResult`. I'm unsure about `ackStatus`, since I'm not sure if we want to only set it when receiving an ack, or always set it before calling the app callback. In this PR I always set it before calling the app callback, so I named it accordingly; if it's supposed to be specific to acks then it can be renamed again